apps/docs/content/guides/auth/server-side/nextjs.mdx

```diff
@@ -626,25 +608,15 @@ Create a `utils/supabase` folder with a file for each type of client. Then copy
 <CH.Code>
 
 ```ts utils/supabase/server-props.ts
-import { createServerClient, type CookieOptions, serialize } from '@supabase/ssr'
+import { createServerClient, serialize } from '@supabase/ssr'
 import { type GetServerSidePropsContext } from 'next'
 
 export function createClient(context: GetServerSidePropsContext) {
   const supabase = createServerClient(
     process.env.NEXT_PUBLIC_SUPABASE_URL!,
     process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!,
     {
-      cookies: {
-        get(name: string) {
```

Comment:
@hf, why were these cookie methods removed? getServerSideProps can access cookies so shouldn't cookie methods be provided here in case the dev wants to check auth, get user info, etc.?

https://nextjs.org/docs/pages/api-reference/functions/get-server-side-props#context-parameter

Reply:
They're not removed, they're replaced with the `getAll` and `setAll` equivalents, because those are the new API.

Reply:
But it looks like they're just replaced with an empty object? See line 617:

```
cookies: {},
```

Apologies if I'm missing something here